src/tag/scss/placeholder.scss

```diff
@@ -20,13 +20,13 @@
     );
 }
 
-%tag-statement-fill {
+%tag-statement-primary {
```

Comment:
renamed to be more semantic in usage. Renaming the variables would require more work so they will stay the same.